influxdb/server.sls

```diff
@@ -35,6 +35,27 @@ influxdb_default:
     - pkg: influxdb_packages
 {%- endif %}
 
+{{server.data.dir}}:
```

Comment:
Sadly no. Give a try, just set a custom path like /mnt/influxdb/data, at least thi sis the case in Ubuntu 16.